ui/v2.5/src/components/Scenes/SceneDetails/SceneVideoFilterPanel.tsx

```diff
@@ -21,316 +55,366 @@ type SliderRange = {
     ) {
-      const feColorMatrix = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feColorMatrix"
-      );
-      feColorMatrix.setAttribute(
-        "values",
-        `${
-          1 +
-          (whiteBalanceValue - whiteBalanceRange.default) /
-            whiteBalanceRange.divider +
-          (redValue - colourRange.default) / colourRange.divider
-        } 0 0 0 0   0 ${
-          1.0 + (greenValue - colourRange.default) / colourRange.divider
-        } 0 0 0   0 0 ${
-          1 -
-          (whiteBalanceValue - whiteBalanceRange.default) /
-            whiteBalanceRange.divider +
-          (blueValue - colourRange.default) / colourRange.divider
-        } 0 0   0 0 0 1.0 0`
-      );
-      videoFilter.appendChild(feColorMatrix);
+      setCurrentOrPreviousVideoStateIsFilter(true);
+      setCurrentOrPreviousVideoStateIsDefaultFilter(false);
+      setFilterFilterValues();
     }
-
-    if (gammaValue !== gammaRange.default) {
-      const feComponentTransfer = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feComponentTransfer"
-      );
-
-      const feFuncR = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feFuncR"
-      );
-      feFuncR.setAttribute("type", "gamma");
-      feFuncR.setAttribute("amplitude", "1.0");
-      feFuncR.setAttribute(
-        "exponent",
-        `${1 + (gammaRange.default - gammaValue) / gammaRange.divider}`
-      );
-      feFuncR.setAttribute("offset", "0.0");
-      feComponentTransfer.appendChild(feFuncR);
-
-      const feFuncG = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feFuncG"
-      );
-      feFuncG.setAttribute("type", "gamma");
-      feFuncG.setAttribute("amplitude", "1.0");
-      feFuncG.setAttribute(
-        "exponent",
-        `${1 + (gammaRange.default - gammaValue) / gammaRange.divider}`
-      );
-      feFuncG.setAttribute("offset", "0.0");
-      feComponentTransfer.appendChild(feFuncG);
-
-      const feFuncB = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feFuncB"
-      );
-      feFuncB.setAttribute("type", "gamma");
-      feFuncB.setAttribute("amplitude", "1.0");
-      feFuncB.setAttribute(
-        "exponent",
-        `${1 + (gammaRange.default - gammaValue) / gammaRange.divider}`
-      );
-      feFuncB.setAttribute("offset", "0.0");
-      feComponentTransfer.appendChild(feFuncB);
-
-      const feFuncA = document.createElementNS(
-        "http://www.w3.org/2000/svg",
-        "feFuncA"
-      );
-      feFuncA.setAttribute("type", "gamma");
-      feFuncA.setAttribute("amplitude", "1.0");
-      feFuncA.setAttribute("exponent", "1.0");
-      feFuncA.setAttribute("offset", "0.0");
-      feComponentTransfer.appendChild(feFuncA);
-
-      videoFilter.appendChild(feComponentTransfer);
+    // Diabling checks because only props.scene is needed to trigger hook
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You really don't want to be disabling `exhaustive-deps`. If you want something to not trigger the hook, use `useRef`.